lib/Model/Listing/AbstractListing.php

```diff
@@ -241,6 +242,35 @@ public function setOrderKey($orderKey, $quote = true)
         return $this;
     }
 
+    /**
+     * @return string
+     */
+    public function getGroupBy()
+    {
+        return $this->groupBy;
+    }
+
+    /**
+     * @param string $groupBy
+     * @param bool $backticks
+     *
+     * @return $this
+     */
+    public function setGroupBy($groupBy, $backticks = true)
+    {
+        $this->setData(null);
+
+        if ($groupBy) {
+            $this->groupBy = $groupBy;
+
+            if (!$backticks || $this->checkIfValueContainBackticks($groupBy)) {
```

Comment:
Actually, this is super inconsistent and should be unified. Guess a TODO for Pimcore 7 would be alright for now?!

Reply:
forget that, you consistent that now :)

Reply:
recommend to remove the $backticks parameter in pimcore 7 with a BC break note